specification/metadata/metadata_schema.md

```diff
@@ -0,0 +1,64 @@
+# FOCUS Metadata Schema 
+Each FOCUS dataset must have a metadata about the schema associated with it. The schema metadata provides information about the structure of the data provided. In the event that the provider modifies the columns provided in the FOCUS dataset, metadata about the schema must be provided, including column definitions.  
+
+## Logical Section: Schema
+
+
+## Available Meta Data:
+ * ### schema_id
+   * Use: unique identifier for the schema
+   * Type: STRING
+ * ### FOCUS_version
+   * Use: the declared version of the FOCUS data within the schema
```

Comment:
Should there be a separate provider version? I will absolutely have multiple versions of 1.0 as the platform evolves.

Reply:
Provider version or data generator version or both?